src/cascadia/Remoting/Monarch.cpp

```diff
@@ -400,6 +450,7 @@ namespace winrt::Microsoft::Terminal::Remoting::implementation
         // After the event was handled, ResultTargetWindow() will be filled with
         // the parsed result.
         const auto targetWindow = findWindowArgs->ResultTargetWindow();
+        const auto& targetWindowName = findWindowArgs->ResultTargetWindowName();

```

Comment:
prefer not using `const auto&` for locals. It _works_, it does what you'd expect, but every time somebody reads it they need to remember that "`const T &` locals are special lifetime-extending references" (and they're no different from `const T` locals)